kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/allegro/OrderDto.java

```diff
@@ -0,0 +1,26 @@
+package com.kodilla.good.patterns.challenges.allegro;
+
+
+public class OrderDto {
+    public User user;
+    public Product product;
+    public boolean isRented;
```

Comment:
Hmm... `isRented`? To miał być sklep, nie zajmuje się raczej wypożyczaniem towaru :)